README.md

```diff
@@ -115,6 +115,7 @@ Software vs DMS build version map
 
 | jwst tag | DMS build | CRDS_CONTEXT |   Date     |          Notes                           |
 | -------- | --------- | ------------ | ---------- | -----------------------------------------|
+|  0.13.8  | B7.3.1    | 0541         | 09/05/2019 | Patch for Build 7.3 released as Build 7.3.1     |
```

Comment:
Will the moderately brain-dead table formatter have a problem with the fact that the end of this line is not aligned with the rest of the table?